test/views/SoftwareChoicesFilterViewSpec.scala

```diff
@@ -194,25 +212,21 @@ class SoftwareChoicesFilterViewSpec extends ViewBaseSpec with SoftwareProvidersT
         }
       }
 
-      "contain a filter for Type of software" should {
+      "contain a filter for Price" should {
 
-        "Suitable for should contain Type of software filter header" in {
-          document.select(Selectors.filterHeader(3)).text shouldBe FilterSearchMessages.typeOfSoftware
+        "Suitable for should contain Price filter header" in {
+          document.select(Selectors.filterHeader(4)).text shouldBe FilterSearchMessages.price
         }
 
-        "Filter Header contains Accounting software" in {
-          document.select(Selectors.accountingFilter).text shouldBe FilterSearchMessages.accountingSoftware
-        }
-
-        "Filter Header contains Connects to spreadsheets" in {
-          document.select(Selectors.spreadsheetsFilter).text shouldBe FilterSearchMessages.spreadSheets
+        "Filter Header contains Free version" in {
+          document.select(Selectors.freeFilter).text shouldBe FilterSearchMessages.free
         }
       }
 
       "contain a filter for Accessibility features" should {
 
         "Suitable for should contain Accessibility features filter header" in {
```

Comment:
Not one of your changes, but the test description is also incorrect here. They could do with updating so they all refer to the right filter if poss.